### ISSUE TITLE
Prevent cube from snapping back to original position

### ISSUE THREAD

**Issue by poisa** [larspetrus/Roofpig#9]
Hi Lars, is there a way to prevent the cube from snapping back to the original rotation after rotating it?

**Comment by larspetrus** [comment]
There isn't. I want to improve that, but it's trickier than it might seem.

My old Java cube would just stay the way you left it, and it was quite hard to get it back to decent looking perspective. So ideally I want it to slide into the right angle, but facing the way you left it. If that makes sense?

But that's a medium sized task that I haven't gotten around to.

Is there anything specific you want to do?

**Comment by poisa** [comment]
Nah, there's nothing specific. I stumbled upon your website many years ago and just for kicks I learned your method and found it really easy to follow. Fast forward a few years, I wanted to show your site to a friend and tried to follow the method again but found that it wasn't as easy to remember each step as it had been years before until I finally realized that I did remember most of the moves but in a different axis. When I tried to play the animations in the orientation that I remembered them I just couldn't.
In fact the only reason I came to see the source for the project was that I thought I might be able to contribute a commit or two with this change but alas it was way over my js abilities (not to mention using coffescript which I have never used before!).
